backend/src/main/scala/cromwell/backend/validation/FailOnStderrValidation.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.backend.validation
+
+import wdl4s.types.{WdlBooleanType, WdlStringType}
+import wdl4s.values.{WdlBoolean, WdlString}
+
+import scalaz.Scalaz._
+
+object FailOnStderrValidation {
+  val key = RuntimeAttributesKeys.FailOnStderrKey
+
+  lazy val instance = new FailOnStderrValidation {}
+
+  lazy val default = instance.withDefault(WdlBoolean(false))
+
+  lazy val optional = default.optional
+
+  val missingMessage =
+    s"Expecting $key runtime attribute to be a Boolean or a String with values of 'true' or 'false'"
+}
+
+trait FailOnStderrValidation extends RuntimeAttributesValidation[Boolean] {
+
+  import FailOnStderrValidation._
+
+  override def key = RuntimeAttributesKeys.FailOnStderrKey
+
+  override def coercion = Seq(WdlBooleanType, WdlStringType)
+
+  override protected def validateValue = {
+    case WdlBoolean(value) => value.successNel
+    case WdlString(value) if value.toLowerCase == "true" => true.successNel
+    case WdlString(value) if value.toLowerCase == "false" => false.successNel
+  }
```

Comment:
hmm looks like we could have a (composable) PF for boolean or Stringified boolean validation